trying to install the tool give error

go: github.com/projectdiscovery/dnsprobe upgrade => v1.0.1
go: github.com/projectdiscovery/retryabledns upgrade => v1.0.4
go: golang.org/x/sys upgrade => v0.0.0-20200420163511-1957bb5e6d1f
go: golang.org/x/crypto upgrade => v0.0.0-20200423211502-4bdfaf469ed5
go: golang.org/x/net upgrade => v0.0.0-20200425230154-ff2c4b7c35a0
github.com/projectdiscovery/dnsprobe/lib
github.com/projectdiscovery/dnsprobe/lib
/home/cypher/Desktop/githubTools/go/pkg/mod/github.com/projectdiscovery/dnsprobe@v1.0.1/lib/dnsprobe.go:41:17: assignment mismatch: 2 variables but "github.com/projectdiscovery/retryabledns".New returns 1 values

--- a/go.mod
+++ b/go.mod
@@ -5,6 +5,6 @@ go 1.14
 require (
 	github.com/miekg/dns v1.1.29
 	github.com/projectdiscovery/gologger v1.0.0
-	github.com/projectdiscovery/retryabledns v1.0.3
+	github.com/projectdiscovery/retryabledns v1.0.4
 	github.com/remeh/sizedwaitgroup v1.0.0
 )

--- a/lib/dnsprobe.go
+++ b/lib/dnsprobe.go
@@ -38,10 +38,7 @@ var DefaultResolvers = []string{
 
 // New creates a dns resolver
 func New(options Options) (*DnsProbe, error) {
-	dnsClient, err := retryabledns.New(options.BaseResolvers, options.MaxRetries)
-	if err != nil {
-		return nil, err
-	}
+	dnsClient := retryabledns.New(options.BaseResolvers, options.MaxRetries)
 
 	return &DnsProbe{dnsClient: dnsClient, questionType: options.QuestionType}, nil
 }

